client/src/shared/services/ajaxFactory.js

```diff
@@ -23,7 +23,7 @@
       });
     };
 
-    ajaxObj.postSignUp = function (user) {
+    ajaxObj.postSignUp = function(user) {
       return $http({
         method: 'POST',
         url: serverUrl + '/signUp',

```

Comment:
this url needs to be '/signup' since the server is looking for that route. use all lowercase